internal/services/migrations/20240609053647_support_natashaskitchen.sql

```diff
@@ -0,0 +1,15 @@
+-- +goose Up
+INSERT INTO websites (host, url)
+VALUES
+    ('chatelaine.com', 'https://chatelaine.com/recipe/salads/mozzarella-clementine-panzanella/?utm_source=whisk&utm_medium=webapp&utm_campaign=fresh_mozzarella_and_clementine_panzanella'),
+    ('downshiftology.com', 'https://downshiftology.com/recipes/shakshuka/?utm_source=whisk&utm_medium=webapp&utm_campaign=shakshuka_recipe_(easy_%26_traditional)'),
+    ('natashaskitchen.com', 'https://natashaskitchen.com/taco-seasoning-recipe/'),
+    ('wellplated.com', 'https://www.wellplated.com/kale-pineapple-smoothie/')
```

Comment:
To avoid having hundreds of migration files related to adding websites, let's move this file to `internal/services/migrations/for-release-v1.2.0`. I'll then merge all these files together when v1.2.0 will be ready, i.e. during https://github.com/reaper47/recipya/issues/319. The URL part of the query in goose up should be the website rather than the recipe, i.e. 

```text
('chatelaine.com', 'https://chatelaine.com'),
('downshiftology.com', 'https://downshiftology.com'),
('natashaskitchen.com', 'https://natashaskitchen.com'),
('wellplated.com', 'https://www.wellplated.com')
```
 
That is actually my bad as I will need to clarify this in the docs.

Regarding Downshifttopology (website 57 when you click the underlined 'supported' in /recipes/add -> Website), it has been added in [this migration](https://github.com/reaper47/recipya/blob/main/internal/services/migrations/20230522154229_websites.sql#L66). It not working is due to a 403, which you fixed by adding the required request headers. You can remove it from this file.

The same can be said for [wellplated.com](https://github.com/reaper47/recipya/blob/main/internal/services/migrations/20230522154229_websites.sql#L265) (website 256).